src/Authoring/DiagnosticTests/DiagnosticTests.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
net5-windows10.0...?

Reply:
It works with both, fwiw; I've updated it